File: apps/dotnet/Simex/Simex/Models/Notification.cs
using System;
using System.Collections.Generic;

namespace Simex.Models;

public partial class Notification
{
    public int Id { get; set; }

    public string Name { get; set; } = null!;

    public string? Description { get; set; }

    public int? UserId { get; set; }

    public int? OperationId { get; set; }

    public int? StateId { get; set; }

    public bool? LogicRemove { get; set; }

    public virtual Operation? Operation { get; set; }

    public virtual NotificationState? State { get; set; }

    public virtual User? User { get; set; }
}

File: apps/web/frontend/src/interfaces/notification/notification.ts
import type { User } from '@interfaces/user/user';
import type { Operation } from '@interfaces/operation/operation';
import type { NotificationState } from './notificationState';

export interface Notification {
  id: number;
  name: string;
  description: string | null;
  user: User;
  operation: Operation;
  state: NotificationState;
  logicRemove: boolean;
}
